pkg/cmd/server/origin/admission/register.go

```diff
@@ -117,24 +117,38 @@ var (
 	)
 )
 
-func init() {
-	admission.PluginEnabledFn = IsAdmissionPluginActivated
-}
-
-func IsAdmissionPluginActivated(name string, config io.Reader) bool {
+func IsAdmissionPluginActivated(name string, config io.Reader) (bool, bool) {
```

Comment:
godoc, the name looks really similar to below.  I'd like to know the diffference.